includes/configure/replace.yaml

```diff
@@ -33,64 +32,29 @@
       dest: "/tmp/{{ vyos_config_temp_config_file }}"
 
   - block:
-    - name: enter configuration mode
-      cli:
-        command: configure
-
-    - name: replace currect running configuration
-      cli:
-        command: "load /tmp/{{ vyos_config_temp_config_file }}"
+    - name: replace current running configuration
+      cli_config:
+        config: "load /tmp/{{ vyos_config_temp_config_file }}"
+        commit_comment: "{{ vyos_config_commit_comment }}"
```

Comment:
commit_comment needs to be checked before using. Since the default value of `vyos_config_commit_comment` is None, the module will have the param commit_comment set to 'None' as a string.

Reply:
@trishnaguha Right. Nice catch. I changed it to `default(omit)`.

Reply:
lgtm